camelot/view/action_steps/update_progress.py

```diff
@@ -117,6 +119,7 @@ class UpdateProgress(ActionStep, DataclassSerializable):
     enlarge: bool = False
     blocking: bool = False
     cancelable: bool = True
+    detail_level: int = logging.INFO # To be determined - we currently map to the loglevels from the logging module
```

Comment:
@tim-vdm deze attribute werd toegevoegd aan de de update progress, om toe te laten de 'visualisatie' van de betreffende detail lijn te 'tunen' ... tbd wat die levels dan moeten zijn ... python log levels zijn op zich nt gewenst, gezien nt descriptief wanneer je de json leest, en hebben (denk ik) ook geen betekenis ?? mss hebben die wel een betekenis, in de zin vd gestandaardiseerde syslog levels ??